project/src/pages/login-page/login-page.tsx

```diff
@@ -0,0 +1,45 @@
+function LoginPage(): JSX.Element {
+  return (
+    <div className="page page--gray page--login">
+      <header className="header">
+        <div className="container">
+          <div className="header__wrapper">
+            <div className="header__left">
+              <a className="header__logo-link" href="main.html">
+                <img className="header__logo" src="img/logo.svg" alt="6 cities logo" width="81" height="41"/>
+              </a>
+            </div>
+          </div>
+        </div>
+      </header>
+
+      <main className="page__main page__main--login">
+        <div className="page__login-container container">
+          <section className="login">
+            <h1 className="login__title">Sign in</h1>
+            <form className="login__form form" action="#" method="post">
+              <div className="login__input-wrapper form__input-wrapper">
+                <label className="visually-hidden">E-mail</label>
+                <input className="login__input form__input" type="email" name="email" placeholder="Email" required={false} />
+              </div>
+              <div className="login__input-wrapper form__input-wrapper">
+                <label className="visually-hidden">Password</label>
+                <input className="login__input form__input" type="password" name="password" placeholder="Password" required={false} />
+              </div>
+              <button className="login__submit form__submit button" type="submit">Sign in</button>
+            </form>
+          </section>
+          <section className="locations locations--login locations--current">
+            <div className="locations__item">
+              <a className="locations__item-link" href="#">
```

Comment:
Чтобы не сыпало ворнингами можно заменить на 
```suggestion
              <a className="locations__item-link" href="/#">
```